packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@griffel/react';
+import {
+  FluentProvider,
+  teamsLightTheme,
+  Body1,
+  Title3,
+  TabList,
+  Tab,
+  Input,
+  Button,
+  Label,
+  Menu,
+  MenuTrigger,
+  MenuList,
+  MenuButton,
+  MenuItemCheckbox,
+  MenuPopover,
+  Slider,
+  Badge,
+  Switch,
+  Radio,
+  RadioGroup,
+  Checkbox,
+} from '@fluentui/react-components';
+import {
+  SearchRegular,
+  bundleIcon,
+  CutRegular,
+  CutFilled,
+  ClipboardPasteRegular,
+  ClipboardPasteFilled,
+  EditRegular,
+  EditFilled,
+  ChevronRightRegular,
+  MeetNowRegular,
+  MeetNowFilled,
+  CalendarLtrFilled,
+  CalendarLtrRegular,
+} from '@fluentui/react-icons';
+
+export interface ContentProps {
+  className?: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    alignItems: 'start',
+    justifyContent: 'center',
+    gridTemplateColumns: '1fr 1fr 1fr',
+    gridTemplateRows: 'auto',
+    gridColumnGap: '5%',
+  },
+  col2: {
+    display: 'grid',
+    gridTemplateRows: '1fr 1fr 1fr',
+    gridRowGap: '10%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  col3: {
+    display: 'grid',
+    gridTemplateColumns: 'auto auto',
+    gridTemplateRows: '1fr 1fr 1fr 1fr',
```

Comment:
```suggestion
    gridTemplateRows: 'repeat(4, 1fr)',
```
other instances below too